jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -134,16 +134,18 @@ jobs:
             DATABASE_URL: postgres://conductor:@localhost:5432/conductor_test
 ```
 
-**Note:** CircleCI 2.0 does not support interpolation of environment variables.
+**Note:**
+CircleCI does not support interpolation of environment variables.
 All defined values are treated literally.
 
-One workaround is to export the required variable within a command.
+If you have bash installed,
```

Comment:
Why be so specific to bash in explaining this?

Reply:
This isn't a workaround for people without bash (i.e. alpine-based images). I wanted to head folks off at the pass before becoming invested in a potential workaround.

Reply:
The same approach works for other shells too, but the specifics differ, e.g. `tcsh` uses `setenv FOO=bar` and from what little knowledge I have, doesn't offer anything similar to `BASH_ENV`, but according to the manpage, `tcsh` will _always_ read `~/.tcshrc` so that's basically the same thing.

Armed with knowledge of their own shell and the specifics of our environment, and our config I suspect most people would work this out themselves. Would we be better off making sure that those aspects are very clearly explained?